src/Client.php

```diff
@@ -154,7 +154,7 @@ function (CommandInterface $req, CommandInterface $res) use ($handle) {
         );
     }
 
-    protected function handleWorkEvent(CommandInterface $command)
+    public function handleWorkEvent(CommandInterface $command)
```

Comment:
Well, during my tests I was facing an issue which I fixed by changing the visibility. But it seems that the issue wasn't linked to the visibility because I haven't been able to reproduce it. Here is my first thoughts :

In the Client class we register the callable 'handleWorkEvent'.
https://github.com/bzikarsky/react-gearman/blob/master/src/Client.php#L52

In EventEmitterTrait, the signature of the function 'on' requires a 'callable' as second argument. To be consider as a valid callable, the array `[$this, 'handleWorkEvent']` must refers to a public function. 
``` php

trait EventEmitterTrait
{
    protected $listeners = [];

    public function on($event, callable $listener)
```